boards/norik/octopus_io_board/dts/bindings/norik,power-controller.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024 Norik Systems
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  This binding is used for controlling battery charger and
+  LDO regulator on the Octopus IO Board.
+
+compatible: "norik,power-controller"
```

Comment:
what is this?

Reply:
Description, but since we did not supply a driver we'll be removing this for now.